### PR TITLE
Update the Makefile message for QCI login

### PR DESCRIPTION
**Pull request by deepsm007**
Message is confusing for the users for QCI login.

/cc @openshift/test-platform

### CODE REVIEW

**Review by bearnl — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -3,8 +3,8 @@ SHELL=/usr/bin/env bash -o errexit
 .PHONY: help check check-boskos check-core check-services dry-core core dry-services services all update template-allowlist release-controllers checkconfig jobs ci-operator-config registry-metadata boskos-config prow-config validate-step-registry new-repo branch-cut prow-config multi-arch-gen message
 
 message:
-	@printf '\e[1mThis Makefile was recently migrated to use QCI as respository of images. Please make sure you are logged in to QCI before executing targets.\e[0m\n'
-	@printf '\e[1mTo login, use your container engine, example for podman: podman login -u=$\(oc --context app.ci whoami) -p=$\(oc --context app.ci whoami -t) quay-proxy.ci.openshift.org\e[0m\n'
+	@printf '\e[1mThis Makefile was recently migrated to use QCI as respository of images. Please make sure you are logged in to QCI before executing targets. Copy the token from https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/ and run `oc registry login`\e[0m\n'
+	@printf '\e[1mTo login, use your container engine, example for podman: podman login -u=$\(oc whoami) -p=$\(oc whoami -t) quay-proxy.ci.openshift.org\e[0m\n'
```
Comment (derekhiggins) [inline Makefile:7]: Can you escape the $'s they don't appear in the output

> To login, use your container engine, example for podman: podman login -u=(oc --context app.ci whoami) -p=(oc --context app.ci whoami -t) quay-proxy.ci.openshift.org

Comment (deepsm007) [reply Makefile:7]: PR merged despite the hold due to race condition

**Pull request merged**